crates/astria-sequencer-relayer/src/relayer/celestia_client/tests.rs

```diff
@@ -1,12 +1,103 @@
-use astria_core::generated::cosmos::base::abci::v1beta1::TxResponse;
+use astria_core::generated::cosmos::{
+    base::abci::v1beta1::TxResponse,
+    tx::v1beta1::{
+        service_client::ServiceClient as TxClient,
+        service_server::{
+            Service,
+            ServiceServer as TxServer,
+        },
+        BroadcastTxRequest,
+        BroadcastTxResponse,
+        GetTxRequest,
+        GetTxResponse,
+    },
+};
 use celestia_types::{
     blob::Commitment,
     nmt::Namespace,
 };
 use prost::bytes::Bytes;
+use tendermint::private_key::Secp256k1 as SigningKey;
+use tokio::net::TcpListener;
+use tonic::{
+    transport::{
+        Endpoint,
+        Server,
+        Uri,
+    },
+    Request,
+    Response,
+    Status,
+};
 
 use super::*;
 
+#[tokio::test(start_paused = true)]
+async fn get_tx_should_timeout() {
```

Comment:
At its core, this tests the timeout inside `Client::get_tx` and thus if the underlying timeout logic of tonic's channel works. This limits the usefulness of this test (because we should always be able to rely on the correct behavior of our building blocks).

Reply:
As discussed, I agree on removing this test, with a view to add black box tests in the near future for sad path cases like gRPC errors, including timeouts.